examples/trials/bert-adaptdl/run_classifier_nni.py

```diff
@@ -590,7 +590,7 @@ def main(params):
                     writer.add_scalar("Accuracy/Valid", stats_eval["accuracy"], epoch)
                     if IS_CHIEF:
                         if epoch < params['num_train_epochs']-1:
-                            nni.report_intermediate_result(stats_eval['accuracy'], stats_eval)
+                            nni.report_intermediate_result(stats_eval['accuracy'], accum=stats_eval)
```

Comment:
@ZeyaWang Our NNI version after this PR would require the key `accum=` while using `report_intermediate_result` for clarity.

Reply:
Thank you for highlighting it.